src/class-parsely.php

```diff
@@ -26,7 +26,7 @@ class Parsely {
 	const CAPABILITY  = 'manage_options';      // The capability required for the user to administer settings.
 
 	/**
-	 * Declare some class propeties
+	 * Declare some class properties
```

Comment:
This is unrelated with the PR, but I found it while working on it and it's not worth a separate PR.